### ISSUE TITLE
Unable to copy paste url from statistics window

### ISSUE THREAD

**Issue by harshanvn** [uBlock-LLC/uBlock#432]
As described in the title, try to copy any url from the statistics window, which spans more than one line. And paste it in the new tab. You see spaces in the url.

This is in firefox. Issue can be reproduced in chrome too..

**Comment by gorhill** [comment]
This CSS recipe works well to chunkify automatically without inserting new line characters:

```
hyphens: none;
word-break:break-all;
word-wrap: break-word;
```

**Comment by gorhill** [comment]
Fixed in 0.8.5.0
